modules/file-rotation/file-rotation-parser.c

```diff
@@ -33,6 +33,7 @@ static CfgLexerKeyword file_rotation_keywords[] =
 {
   { "file_rotation", KW_FILE_ROTATION },
   { "size", KW_SIZE },
+  { "interval", KW_INTERVAL },
```

Comment:
Adding the interval option does not belong to the flush-signal commit.

Reply:
I think these changes have been lost due to force push, what do you suggest should I separate out the commits ?

Reply:
Yes, please create a separate commit.

As I mentioned:
We encourage faking commit history while working on a feature branch.
The goal here is to create a nice linear patchset for your pull request, that is easy to review.
The commit message should describe the change it is containing, which needs to be small/atomic, concentrating on one single thing.

Reply:
Updated :+1: